poonam/JS/Assignment13_js/que3.html

```diff
@@ -7,20 +7,25 @@
         <title>Previous date</title>
     </head>
     <body>
+        Enter Date: <input type="date" id="inputDate">
+        Enter no. of days: <input type="number" id="inputDays">
+        <button onclick="output()">Submit</button>
         <script>
-            var input = prompt("Enter date of birth in the following format:\nMonth, date, year ");
-            var date = new Date(input);
-            var days = prompt("Enter days: ");      
-            document.write("Date: " + date)          
-            function getDateAgo(date, days) {
-            let dateCopy = new Date(date);
-                                                                                                                                                
-            dateCopy.setDate(date.getDate() - days);
-            return dateCopy.getDate();
-            }                                                                                                                                  
-            document.write("<br>Yesterday's date: " + getDateAgo(date, 1) );
-            document.write("<br>Day before yesterday's date: " + getDateAgo(date, 2) );
-            document.write("<br>Date: " + getDateAgo(date, 365) );
+            function output() {   
+                var input = document.getElementById("inputDate").value;
+                var days = parseInt(document.getElementById("inputDays").value); 
+                var date = new Date(input);  
+                document.write("Date: " + date)          
+                function getDateAgo(date, days) {
+                let dateCopy = new Date(date);
+                                                                                                                                                    
+                dateCopy.setDate(date.getDate() - days);
+                return dateCopy.getDate();
+                }                                                                                                                                  
+                document.write("<br>Previous date: " + getDateAgo(date, days) );
+                document.write("<br>Day before previous date: " + getDateAgo(date, days+1) );
+                document.write("<br>Date: " + getDateAgo(date, 365) );
+            }
         </script>
     </body>
 </html>
```

Comment:
this program is still not working poonam, take someone's help...or check qadir or sanket's program

Reply:
mam but it is working on my side